docker/1.15/Dockerfile.cpu

```diff
@@ -4,6 +4,7 @@ LABEL maintainer="Amazon AI"
 # Specify LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
 # https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
```

Comment:
Looks like this label was added only to `docker/1.15/Dockerfile.cpu` and `docker/2.1/Dockerfile.cpu`. Was that intentional?

Reply:
Yes, this is intentional. Only the latest 1.x and 2.x versions are supporting multi-model mode.